### PR TITLE
 net-im/telegram-desktop-bin: version bump and cleanup

### PR DESCRIPTION
**Pull request by henning-schild**

### CODE REVIEW

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg
```
Comment (juippis) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.6.0.ebuild]: gnome2-utils isn't needed for icon_cache_update anymore, it's included in xdg-utils. I see you inherit xdg and call for the default phase setups, if you update EAPI to 7, those should now also include xdg_icon_cache_update.

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+QA_PREBUILT="usr/lib/${PN}/Telegram"
+
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/gobject-introspection
+	>=sys-apps/dbus-1.4.20
+	x11-libs/libX11
+	>=x11-libs/libxcb-1.10[xkb]
+	>=media-libs/fontconfig-2.13
```
Comment (juippis) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.6.0.ebuild]: Please keep it sorted (media-libs < sys-apps)

**Comment by henning-schild** [comment]
4e939ba4705b0e1fab7a2305312a93cc47e0364b

**Comment by juippis** [comment]
Soo... do you two work it out together? Or should @Zlogene be added as co-maintainer so he sees when you try to update it?

**Comment by henning-schild** [comment]
I have no clue and was not in touch with @Zlogene . Just created yet another bump, maybe the review results can go in there. For now it is a simple copy, like that one was. #11491

**Pull request closed without merging**